chia/wallet/util/debug_spend_bundle.py

```diff
@@ -40,7 +40,7 @@ def dump_coin(coin: Coin) -> str:
     return disassemble(coin_as_program(coin))
 
 
-def debug_spend_bundle(spend_bundle, agg_sig_additional_data=bytes([3] * 32)) -> None:
+def debug_spend_bundle(spend_bundle, agg_sig_additional_data=bytes([3] * 32)) -> str:
```

Comment:
Rather than use the `str += more_text` pattern, I'd suggest rewriting as `print(..., file=f)` and then pass in `f` as a file here.

Reply:
You could be even more lazy and define a new local `print` function that uses the built-in one:

```
def debug_spend_bundle(spend_bundle, agg_sig_additional_data=bytes([3] * 32), f=sys.stdin) -> str:

    def print(*args, **kwargs):
        __builtins__.print(*args, **kwargs, f=f)
```

And now you don't need any of the changes below.

Reply:
haha, nice